test/test_facebook.py

```diff
@@ -197,5 +197,40 @@ def test_parse_signed_request_when_correct(self):
         self.assertTrue('algorithm' in result)
 
 
+class TestAPIRequest(FacebookTestCase):
+    def test_request_args_by_none(self):
+        FB_VER = 2.7
+        FB_OBJECT_ID = "1846089248954071_1870020306560965"
+        token = facebook.GraphAPI().get_app_access_token(
+            self.app_id, self.secret)
+        graph = facebook.GraphAPI(access_token=token, version=FB_VER)
+
+        # args=None
+        self.assertRaises(
+            TypeError, graph.request, FB_OBJECT_ID, args=None, post_args=None)
```

Comment:
I am not sure I understand what this method is supposed to be testing. If the structure of the `request` method changes, `None` could become a valid argument. This would not necessarily mean that the `request` method was broken, though.